File: java/Two_sum.java
package java;
class Solution {
    public int[] twoSum(int[] nums, int target) {
    
       int array_length=nums.length;
        for(int i=0;i<array_length;i++)
        {
            for(int j=i+1;j<array_length;j++)
            {
                if(nums[i]+nums[j]==target){
               System.out.println("the indices which add upto target is " + i + ", " + j);
               return new int[]{i,j};
        }
            }
        
        }
     return new int[]{};
}
}

File: python/Two_sum.py
#this is for leetcode so no need to take input from user
#sum of two indices in a list that equals a target number

class Solution(object):
    def twoSum(self, nums, target):
        """
        :type nums: List[int]
        :type target: int
        :rtype: List[int]
        """

        for i in range(len(nums)):  
            for j in range(i + 1, len(nums)):
                if nums[i] + nums[j] == target:
                    return [i, j]


        return []